src/sub-clients/accounts/client/FakeAccountsClient.ts

```diff
@@ -203,19 +203,20 @@ export class FakeAccountsClient implements AccountsClient, Clearable {
     );
     const accountToInsert = {
       ...account,
-      name: updateRequest?.name || account.name,
-      manager: updateRequest?.manager || account.manager,
-      status: updateRequest?.status || account.status,
-      domains: updateRequest?.domains || account.domains,
-      deliveryMethod: updateRequest?.deliveryMethod || account.deliveryMethod,
+      name: updateRequest.name || account.name,
+      manager: updateRequest.manager || account.manager,
+      status: updateRequest.status || account.status,
+      domains: updateRequest.domains || account.domains,
+      deliveryMethod: updateRequest.deliveryMethod || account.deliveryMethod,
       defaultLicenseDurationYears:
-        updateRequest?.defaultLicenseDurationYears ||
-        account.defaultLicenseDurationYears,
+        updateRequest.defaultLicenseDurationYears !== undefined
+          ? updateRequest.defaultLicenseDurationYears
+          : account.defaultLicenseDurationYears,
```

Comment:
We were actually working on this at the same time, and found the nullish coalescing operator was a nice option here. Does the null and undefined check in one. More info on it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
```suggestion
        updateRequest.defaultLicenseDurationYears ?? account.defaultLicenseDurationYears,
```